docs/getting-started/walk-through/advanced-hera-features.md

```diff
@@ -89,12 +89,30 @@ conflicts.
 ## Experimental Features
 
 From time to time, Hera will release a new feature under the "experimental feature" flag while we develop the feature
-and ensure stability. Currently this is used for the `RunnerScriptConstructor` seen in the
-[runner script example](../../examples/workflows/callable_script.md).
+and ensure stability. 
 
 To enable experimental features you must set the feature by name to `True` in the `global_config.experimental_features`
 dictionary before using the feature:
 
+```py
+global_config.experimental_features["NAME_OF_FEATURE"] = True
+```
+
+### Currently supported experimental features:
+
+#### `RunnerScriptConstructor`
+What is this used for? [runner script example](../../examples/workflows/callable_script.md)
```

Comment:
```suggestion
The `RunnerScriptConstructor` found in `hera.workflows.script` and seen in the
[callable script example](../../examples/workflows/callable_script.md) is a robust way to run Python functions on Argo.
The image used by the script should be built from the source code package itself and its dependencies, so that the
source code's functions, dependencies, and Hera itself are available to run. The `RunnerScriptConstructor` is also
compatible with Pydantic so supports deserializing inputs to Python objects and serializing outputs to json strings. It
must be enabled with the `script_runner` feature flag as below.
```